pyramid/scripts/pcreate.py

```diff
@@ -99,9 +106,13 @@ def output_path(self):
     def project_vars(self):
         output_dir = self.output_path
         project_name = os.path.basename(os.path.split(output_dir)[1])
-        pkg_name = _bad_chars_re.sub(
-            '', project_name.lower().replace('-', '_'))
-        safe_name = pkg_resources.safe_name(project_name)
+        if self.options.package_name is None:
+            pkg_name = _bad_chars_re.sub(
+                '', project_name.lower().replace('-', '_'))
```

Comment:
Sorry I read that test wrong, I thought it was `is not None`. So are we not validating the package name if you specify `--package-name`?